spec/person_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rspec'
+require './book'
+require './rental'
+require './person'
+describe Person do
+  before(:each) do
+    @person = Person.new(20, 'John')
+  end
+  context 'when creating a new book' do
```

Comment:
- [ ] Please don't forget to update the text you're not creating a new book here! 😅